pkg/cmd/render/render.go

```diff
@@ -199,3 +202,47 @@ func (r *renderOpts) Run() error {
 
 	return nil
 }
+
+func featureGateManifests(o genericrenderoptions.GenericOptions) (genericrenderoptions.RenderedManifests, error) {
+	if len(o.RenderedManifestInputFilenames) == 0 {
+		return nil, fmt.Errorf("cannot return FeatureGate without rendered manifests")
+	}
+
+	inputManifest, err := o.ReadInputManifests()
+	if err != nil {
+		return nil, fmt.Errorf("error reading input manifests: %w", err)
+	}
+	featureGates := inputManifest.ListManifestOfType(configv1.GroupVersion.WithKind("FeatureGate"))
+	if len(featureGates) == 0 {
+		return nil, fmt.Errorf("no FeatureGates found in manfest dir: %v", o.RenderedManifestInputFilenames)
+	}
+
+	ret := genericrenderoptions.RenderedManifests{}
+
+	var prev *genericrenderoptions.RenderedManifest
+	var featureGate *configv1.FeatureGate
+	for i := range featureGates {
```

Comment:
Why not use `GetManifest()` introduced in https://github.com/openshift/library-go/pull/1516? Seems it would provide the same functionality since we know `FeatureGate` is a singleton in the cluster